ipl-auction-react/src/control/outLinedTextField.component.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classes from "./outLinedTextField.css";
+
+const OutLinedTextField = (props) => {
+  const {
+    id,
+    type,
+    value,
+    checked,
+    disabled,
+    onChange,
+    fieldName,
+    customizeClassNameForInput,
+    customizeClassNameForLabel,
+    isClassNameProvidedForInput,
+    isClassNameProvidedForLabel,
+  } = props;
+  return (
+    <div className={classes.textfield}>
+      <input
+        className={
+          isClassNameProvidedForInput
```

Comment:
Instead of getting isClassNameProvidedForInput can we do with customizeClassNameForInput? 
For all label also..

Reply:
yes, we can get only customizeClassNameForInput for changing the input CSS style